server/auth/simple_token.go

```diff
@@ -156,6 +156,13 @@ func (t *tokenSimple) invalidateUser(username string) {
 }
 
 func (t *tokenSimple) enable() {
+	t.simpleTokensMu.Lock()
+	if t.simpleTokenKeeper != nil { // already enabled
+		t.simpleTokensMu.Unlock()
+		return
+	}
+	defer t.simpleTokensMu.Unlock()
```

Comment:
Thanks @cfz one comment here - I think you should put the defer statement before the if loop and remove the Unlock statement from the if loop. Thanks! cc @mitake

Reply:
good comment @spzala. updated

Reply:
@cfz thanks for addressing it quickly! A nit now - can you please remove the blank lines on 161 and 165 for better readability. Thanks!

Reply:
done @spzala